<launch>
 <group ns="lidar1">
  <node name="lidar01" pkg="bluesea2" type="bluesea2_node" output="screen" >
    #ROS#
    <param name="topic" value="scan"/>
    <param name="frame_id" value="map" />
     #DATA#
    <param name="raw_bytes" value="3"/>
    <param name="min_dist" value="0.1"/>
    <param name="max_dist" value="50.0"/>
    <param name="from_zero" value="false"/>
    <param name="output_scan" value="true" />
    <param name="output_cloud" value="false"/>
    <param name="output_360" value="true" />
    <param name="inverted" value="true"/>
    <param name="reversed" value="false"/>
    <param name="hard_resample" value="false"/>
    <param name="soft_resample" value="false"/>
    <param name="with_angle_filter" value="false"/>
    <param name="min_angle" value="-3.1415926"/>
    <param name="max_angle" value="3.1415926"/> 
    <rosparam param="mask1" >[1.57,3.14]</rosparam>
    <rosparam param="mask2" >[-1,0]</rosparam> 
    <param name="error_circle" value="3"/>
    <param name="error_scale" value="0.9"/>
    #CONNECT#
    <param name="type" value="udp" />
    <param name="lidar_port" value="6543" />
    <param name="local_port" value="6668" />
    <param name="lidar_ip" value="192.168.158.98"/>
    <param name="group_listener" value="false" />
    <param name="group_ip" value="224.0.0.11" />
    #GET#
    <param name="uuid" value="-1" />
    #SET#
    <param name="rpm" value="-1"/>
    <param name="resample_res" value="-1"/>
    <param name="with_smooth" value="-1" />
    <param name="with_deshadow" value="-1" />
    <param name="alarm_msg" value="-1"/>
    <param name="direction" value="-1"/>
  </node>
 </group>
</launch>
